packages/gatsby/src/query/file-parser.js

```diff
@@ -418,7 +418,8 @@ export default class FileParser {
       return null
     }
 
-    if (!text.includes(`graphql`)) return null
+    if (!text.includes(`graphql`) && !text.includes(`gatsby-plugin-image`))
```

Comment:
Can you add a comment here why we have to add this plugin cause that's not 100% clear to me